webviz_config/webviz_factory.py

```diff
@@ -0,0 +1,11 @@
+class WebvizFactory:
+    """Base class for all factories that want to register themselves in
+    WebvizFactoryRegistry.
+    """
```

Comment:
Should this also be an ABC? I.e. it doesn't make sentence to create an instance of `WebvizFactory` directly?

Reply:
I did consider it, but I don't think it makes sense since there are currently no abstract methods. I believe that it is the presence of actual abstract methods that make the class abstract and therefore prohibits instantiation. In that case I don't think it helps making the class itself an ABC. Not sure though...